hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesAccessor.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2020-2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app;
+
+import com.hedera.node.app.service.consensus.ConsensusService;
+import com.hedera.node.app.service.contract.ContractService;
+import com.hedera.node.app.service.file.FileService;
+import com.hedera.node.app.service.freeze.FreezeService;
+import com.hedera.node.app.service.network.NetworkService;
+import com.hedera.node.app.service.schedule.ScheduleService;
+import com.hedera.node.app.service.token.CryptoService;
+import com.hedera.node.app.service.token.TokenService;
+import com.hedera.node.app.service.util.UtilService;
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * A {@code ServiceAccessor} is used to pass all services to components via a single parameter.
+ *
+ * @param consensusService a {@link ConsensusService}
+ * @param contractService a {@link ContractService}
+ * @param cryptoService a {@link CryptoService}
+ * @param fileService a {@link FileService}
+ * @param freezeService a {@link FreezeService}
+ * @param networkService a {@link NetworkService}
+ * @param scheduleService a {@link ScheduleService}
+ * @param tokenService a {@link TokenService}
+ * @param utilService a {@link UtilService}
+ */
+public record ServicesAccessor(
+        @Nonnull ConsensusService consensusService,
```

Comment:
In poc there is another class called `ServicesContext`, we are not using it here ?

Reply:
This is a part, that I do not understand in the poc. When do you want to pass around a service together with `StateRegistryImpl`? My feeling is, that this is a left-over from an earlier implementation. But I need to ask Richard.

Reply:
As recently discussed, `ServicesContext` is not needed in its current form.